src/main/java/internet/store/dao/UserDao.java

```diff
@@ -1,6 +1,8 @@
 package internet.store.dao;
 
 import internet.store.model.User;
+import java.util.Optional;
 
 public interface UserDao extends GenericDao<User, Long> {
+    Optional<User> findByLogin(String logIn);
```

Comment:
```suggestion
    Optional<User> findByLogin(String login);
```
everywhere - login is a single word